src/asana/controller.py

```diff
@@ -38,6 +38,29 @@ def update_task(pull_request: PullRequest, task_id: str):
     maybe_complete_tasks_on_merge(pull_request)
 
 
+def create_review_subtask(pull_request: PullRequest, parent_task_id: str, reviewer_handle: str):
+    asana_id = asana_helpers.asana_user_id_from_github_handle(reviewer_handle)
+    task_name = asana_helpers.subtask_name_from_pull_request(pull_request)
+    task_description = asana_helpers.subtask_description_from_pull_request(
+        pull_request, reviewer_handle
+    )
+    due_date_str = asana_helpers.default_due_date_str()
+    return asana_client.create_subtask(
+        parent_task_id, asana_id, task_name, task_description, due_date_str=due_date_str
+    )
+
+
+def update_subtask(pull_request: PullRequest, subtask_id):
```

Comment:
The naming here is a bit confusing, perhaps just `reopen_subtask_if_completed` would be more accurate?